hevc/hevcdecoderconfigurationrecord.go

```diff
@@ -212,3 +212,8 @@ func (h *DecConfRec) GetNalusForType(naluType NaluType) [][]byte {
 	}
 	return nil
 }
+
+// AddNaluArrays add new nalus to HEVCDecConfRec
```

Comment:
This project is starting to follow the documentation strategy and have a full sentence including period, so it should rather be "AddNaluArrays appends new nalus to HEVCDecConfRec.".

Reply:
Good to know that, I updated it